open-ce/env_config.py

```diff
@@ -17,18 +18,46 @@
           " for a list of requirements.")
     sys.exit(1)
 
+class Key: #pylint: disable=no-init,too-few-public-methods
+    '''Enum for Env Config Keys'''
+    IMPORTED_ENVS = 0
+    CHANNELS=1
+    PACKAGES=2
+    GIT_TAG_FOR_ENV=3
+    GIT_TAG=4
+    FEEDSTOCK=5
+
+ENV_CONFIG_KEYS = {
+    Key.IMPORTED_ENVS: "imported_envs",
+    Key.CHANNELS: "channels",
+    Key.PACKAGES: "packages",
+    Key.GIT_TAG_FOR_ENV: "git_tag_for_env",
+    Key.GIT_TAG: "git_tag",
+    Key.FEEDSTOCK: "feedstock"
+}
+
+_PACKAGE_SCHEMA ={
+    ENV_CONFIG_KEYS[Key.FEEDSTOCK]: utils.make_schema_type(str, True)
```

Comment:
There should also be a 'recipes' field here that can be a list of strings.

Reply:
Key.CHANNELS can be in both the _ENV_CONFIG_SCHEMA and the _PACKAGE_SCHEMA

Reply:
Right now all of the CE files only have `feedstock` under `packages`.

Reply:
Fixed.